README.md

```diff
@@ -1,2 +1,150 @@
+
+### Project Mechanical design
+ ![Project Images](https://github.com/Nikhil-Choudhary/VR-Flight-Simulator/blob/master/Photos%20And%20Videos/photos/Project%20Mechanical%20design.jpeg)
+
+### Game made on unity
+![Project Images](https://github.com/Nikhil-Choudhary/VR-Flight-Simulator/blob/master/Photos%20And%20Videos/photos/Game.jpeg)
+***
+## ABSTRACT:
+
+The project we made is known as vr flight simulator.It is a simulation of real life plane experience.
+In this project We had used a ultrasonic sensor, a bump sensor, a potentiometer sensor, a gyroscope
+sensor (esp8266), and a esp9250 module to make a joystick and a throttle handle. The joytick physically
+moved in vertical circular motion in left write direction, and it moved front and backwards producing 
+respective motion in game. The throttles went front and backwards to accelerate and decelerate the plane
+respectively.A game on pc was developed by us which contains a plane and some objects which can be shot.
+***
+## MOTIVATION:
+
+Our motivation for doing this project was the creativity of the idea and our fascination with the end result.
+***
+## MECHANICAL ASPECT OF DESIGN:
+
+Our mechanical parts consists of
+
+1. joystick
+2. throttle
+
+Explaination :-
+	
+1. joystick
+	
+	this device is used for controlling the aeroplane so that user experiences reality ( gives us a real experience ), with the help of this device we tried to make environment like that of aeroplane .
+	
+2. throttle 
+	
+	this device is basically used for controling the speed of aeroplane.
+	
+Both the devices are made from basic components such as ply , a man made springs , steel channels , ball bearings , and joining components
+***
+## Electronic ASPECT OF DESIGN:
+
+Electronics Consist of use of
+
+1. Ultrasonic sensor
+
+	for giving pitch to plane
+	
+2. Gyro sensor
+
+	for detecting right-left motion of plane
+
+3. Potentiometer
+
+	for detecting throttle
+	
+4. Bump Sensor
+
+	for detecting Rocket shooting
+
+5. Esp module
+
+	for receiving data from sensors and sending it to unity	
+***
+## Complete Working of Project
+
+what we are doing is that
+* taking input from 4 sensors(in form of nos.) and sending this input to unity software through esp module
+* after receiving data we are scaling it in unity to make it in suitable form and sending it to our game
+* then we are using a software called trinus vr to project computer screen onto our mobile in form of 3d environment by making pc as server and mobile as receiver
+* This is it. Syncing sensor input and input in unity was very crucial.
+ ***
+## COST STRUCTURE:
+
+1. Mechanical 
+	* welding : rs 70
+	* ply     : rs 300
+	* wire    : rs 200
+ 
+2. Electrical
+	* esp8266 module       : rs 500
+	* ultrasonic sensor    : rs 100
+	* bump sensor          : rs 20
+	* potentiometer sensor : rs 50
+***
+## APPLICATIONS:
+
+It can used for real life pilot training. It can also be used as a game for mass selling. 
```

Comment:
Write this as a numbered List.
1. Can be used for real life pilot training.
2. Can be used for mass selling.